addons/sourcemod/scripting/mce/internal_functions.inc

```diff
@@ -59,15 +59,16 @@ NominateResult InternalNominateMap(char[] map, bool force, int owner)
 
 		RemoveFromArray(g_NominateList, 0);
 		RemoveFromArray(g_NominateOwners, 0);
-		g_sNominations[owner] = "";
+		ClearStoredNomination(owner_);
 
 		if (owner_ == 0)
 			g_NominateReservedCount--;
 		else
 			g_NominateCount--;
+
+		strcopy(g_sNominations[owner], sizeof(g_sNominations[]), map);
```

Comment:
i dont think its supposed to be moved here since its just the last step that nominated the map